### ISSUE TITLE
Fail to build on aarch64, devuan ceres.

### ISSUE THREAD

**Issue by clort81** [SimonN/LixD#426]
Fail to build on aarch64, devuan ceres.

```
dub -f
Performing "debug" build using /usr/bin/ldc2 for aarch64, arm_hardfloat.
allegro 4.0.4+5.2.0: building configuration "no-libs"...
derelict-util 3.0.0-beta.2: building configuration "library"...
derelict-enet 4.2.0: building configuration "library"...
enumap 0.4.2: building configuration "library"...
bolts 1.3.1: building configuration "library"...
optional 1.3.0: building configuration "library"...
taggedalgebraic 0.11.22: building configuration "library"...
sdlang-d 0.10.6: building configuration "library"...
lix 0.9.39: building configuration "application"...
Compiling Lix 0.9.39 with LDC for Linux, 64-bit...
/usr/lib/ldc/aarch64-linux-gnu/include/d/std/algorithm/comparison.d(1531,6): Warning: skipping definition of function `std.algorithm.comparison.max!(int, int).max` due to previous definition for the same mangled name: _D3std9algorithm10comparison__T3maxTiTiZQjFNaNbNiNfiiZi
Error: warnings are treated as errors
       Use -wi if you wish to treat warnings only as informational.
/usr/bin/ldc2 failed with exit code 1.
```
How can i build LixD?

**Comment by SimonN** [comment]
Hi!

Use the same workaround as in https://github.com/SimonN/LixD/issues/396#issuecomment-991943232: In your dub.json, add this line:

    "buildRequirements": ["allowWarnings"],

This will make the Lix build pass despite the warning in Phobos.

I'm about to release Lix 0.9.40 tomorrow morning that contains this fix.

**Comment by clort81** [comment]
Thank you. it's using only 30% cpu in a huge window on my little ARM sbc (cortex a73).
This is wonderful.

**Comment by SimonN** [comment]
Thanks for the feedback! Always great to hear when Lix runs well on a new platform. Enjoy!

**Comment by SimonN** [comment]
This should be fixed in 0.9.40 where I've included `"buildRequirements": ["allowWarnings"],` by default.

If you encounter any more build errors, reopen this or file a new bug.
